README.md

```diff
@@ -57,3 +57,6 @@ option java_package = "io.numaproj.numaflow.function.v1";
 ```protobuf
 option java_package = "io.numaproj.numaflow.sink.v1";
 ```
+
+## Code Style
+If you use IntelliJ as Java IDE, use [google style guide](https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml). 
```

Comment:
we support editorConfig. Do we need this?

Reply:
Oh, I didn't know about editorConfig. Could you elaborate more on how to use it? Would like to ensure multiple developers follow the same coding style for this package.

Reply:
https://www.jetbrains.com/help/idea/editorconfig.html#:~:text=style%20to%20an%20.-,editorconfig%20file%EF%BB%BF,of%20the%20IDE%2Dlevel%20schemes.&text=%2C%20select%20Export%20and%20then%20EditorConfig%20File.

Reply:
editorConfig file is present in the project's root directory, you just need to point the ide to use that while formatting..

Reply:
Thanks! Will do. +1